xknxproject/loader/application_program_loader.py

```diff
@@ -78,8 +78,8 @@ def load(
                             ),
                             datapoint_type=datapoint_type,
                         )
-                if elem.tag.endswith("Static"):
-                    # we don't need anything after Static tag (Dynamic, Languages)
+                if elem.tag.endswith("Dynamic"):
+                    # We don't need anything after dynamic
```

Comment:
We should stop when reaching the end of `ApplicationPrograms` currently.
`Dynamic` (which we don't need), as well as `ModuleDefs` (which we need), both are optional and come after `Static`. If we had an application without a `Dynamic` block we would parse `Languages` (translations) as well with the current implementation - which we don't need.